packages/database/src/migrations/20240822044835-addLoginAttemptsTable-modifies-schema.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const TABLE_NAME = 'login_attempts';
+
+exports.up = async function (db) {
+  await db.runSql(
+    `CREATE TABLE IF NOT EXISTS ${TABLE_NAME} (
```

Comment:
this comes from the node-rate-limiter docs https://github.com/animir/node-rate-limiter-flexible/wiki/PostgreSQL